src/app/pages/MapPage/MapPage.tsx

```diff
@@ -1,77 +1,61 @@
 import { TileLayer } from '@datapunt/react-maps'
-import { NonTiledLayer } from '@datapunt/arm-nontiled'
+// import { NonTiledLayer } from '@datapunt/arm-nontiled'
+import { BaseLayer, constants, Map, mapPanelComponents, useStateRef } from '@datapunt/arm-core'
 import { MarkerClusterGroup } from '@datapunt/arm-cluster'
-import DrawTool from './Components/DrawTool'
+import { PositionPerSnapPoint } from '@datapunt/arm-core/es/components/MapPanel/constants'
 import GeoJSON from './Components/GeoJSON'
-import MapPanelContainer from './MapPanelContainer'
-import MAP_CONFIG from '../../../map/services/map.config'
 import MapContext from './MapContext'
+import ViewerContainer from './Components/ViewerContainer'
+import PointSearchMarker from './Components/PointSearchMarker'
+import { Overlay, SnapPoint } from './Components/types'
+import PointSearchResults from './Components/PointSearchResults'
+import MapLegend from './Components/MapLegend'
 import handleMapClick from './utils/handleMapClick'
-import MapPreviewPanelContainer from './MapPreviewPanelContainer'
+import MAP_CONFIG from '../../../map/services/map.config'
+
+const { MapPanel, MapPanelDrawer, MapPanelProvider } = mapPanelComponents
+const { DEFAULT_AMSTERDAM_MAPS_OPTIONS } = constants
 
 const StyledMap = styled(Map)`
   width: 100%;
-  height: calc(100% - 50px);
-  top: 50px;
-`
-
-const BottomLeftHolder = styled.div`
-  display: flex;
+  height: 100%;
 `
 
 const MapView = styled.div`
   height: calc(100% - 50px);
-  top: 50px;
-`
-
-const StyledViewerContainer = styled(ViewerContainer)`
-  z-index: 400;
-  height: calc(100% - 50px);
-  top: 50px;
-`
-
-// This can be deleted once we use the new MapDrawer / MapPanel
-const MapPanelContainerWrapper = styled.div`
-  bottom: 20px;
-  margin-right: ${themeSpacing(2)};
-
-  & > section {
-    left: 0;
-    bottom: 0;
-    position: relative;
-    max-height: 80vh;
-  }
+  position: relative;
 `
 
-type extraLayerTypes = {
+type MarkerGroup = {
   id: string
-  editing: {
-    _enabled: boolean
-    disable: () => void
-  }
+  markers: LatLngTuple[]
 }
 
-type PolygonType = extraLayerTypes & Polygon
-type PolylineType = extraLayerTypes & Polyline
-
-export type ExtendedLayer = PolygonType | PolylineType
+const GlobalStyle = createGlobalStyle`
+  body {
+    touch-action: none;
+    overflow: hidden; // This will prevent the scrollBar on iOS due to navigation bar
+  }
+`
 
-type MarkerGroup = {
-  id: string
-  markers: LatLngTuple[]
+export const MAP_PANEL_SNAP_POSITIONS: PositionPerSnapPoint = {
+  [SnapPoint.Full]: '40%',
+  [SnapPoint.Halfway]: '40%',
+  [SnapPoint.Closed]: '30px',
 }
 
 const MapPage: React.FC = () => {
   const [isLoading, setIsLoading] = useState(false)
   const [showDrawTool, setShowDrawTool] = useState(true)
-  const [mapInstance, setMapInstance] = useState<L.Map>()
   const [markerGroups, setMarkerGroups, markerGroupsRef] = useStateRef<MarkerGroup[]>([])
-
+  const [mapInstance, setMapInstance] = useState<L.Map>()
```

Comment:
Looks like `mapInstance` is never used here. We can omit it:
```suggestion
  const [, setMapInstance] = useState<L.Map>()
```